internal/pkg/container/kernel.go

```diff
@@ -0,0 +1,47 @@
+package container
+
+import (
+	"path"
+	"path/filepath"
+	"sort"
+
+	"github.com/hpcng/warewulf/internal/pkg/wwlog"
+)
+
+var (
+	kernelSearchPaths = []string{
+		// This is a printf format where the %s will be the kernel version
+		"/boot/vmlinuz-*",
+		"/lib/modules/*/vmlinuz*",
```

Comment:
Kernel location probably should have a commandline override so we can specify the kernel location: see the discussion here https://github.com/hpcng/warewulf/issues/156

Reply:
The kernel location doesn't matter as long as it is in a standard location. Are you asking for kernels that aren't in standard locations?

Reply:
It looks like this isn't an issue any more -- so I think the defaults are fine... and if they don't work, then I can always move my kernels within a container and re-import :)